cliquet/resource/viewset.py

```diff
@@ -153,7 +153,9 @@ def is_endpoint_enabled(self, endpoint_type, resource_name, method,
         Uses the settings to tell so.
         """
         readonly_enabled = asbool(settings.get('readonly'))
-        if readonly_enabled and method not in self.readonly_methods:
+        readonly_method = method.lower() in [m.lower() for m in
+                                             self.readonly_methods]
+        if readonly_enabled and not readonly_method:
```

Comment:
why the "not" here? I would have naively expected it to return False when the current method is readonly.

e.g.

```python
if readonly_enabled and readonly_method:
    return False
```